cclib/parser/nwchemparser.py

```diff
@@ -1077,10 +1095,146 @@ def extract(self, inputfile, line):
                 while line.strip():
                     tokens = line.split()
                     assert len(tokens) == 8
-                    atomcoords_step.append([float(c) for c in tokens[2:5]])
+                    atomcoords_step.append([utils.float(c) for c in tokens[2:5]])
                     line = next(inputfile)
                 self.atomcoords.append(atomcoords_step)
 
+        # Extract Thermochemistry in au (Hartree)
+        #
+        # have to deal with :
+        # Temperature                      =   298.15K
+        # frequency scaling parameter      =   1.0000
+        # Zero-Point correction to Energy  =  259.352 kcal/mol  (  0.413304 au)
+        # Thermal correction to Energy     =  275.666 kcal/mol  (  0.439302 au)
+        # Thermal correction to Enthalpy   =  276.258 kcal/mol  (  0.440246 au)
+        # Total Entropy                    =  176.764 cal/mol-K
+        # - Translational                =   44.169 cal/mol-K (mol. weight = 448.1245)
+        # - Rotational                   =   37.018 cal/mol-K (symmetry #  =        1)
+        # - Vibrational                  =   95.577 cal/mol-K
+        # Cv (constant volume heat capacity) =  103.675 cal/mol-K
+        # - Translational                  =    2.979 cal/mol-K
+        # - Rotational                     =    2.979 cal/mol-K
+        # - Vibrational                    =   97.716 cal/mol-K
+        if line[1:12] == "Temperature":
+            self.set_attribute('temperature', utils.float(line.split()[2][:-1]))

```

Comment:
A lot of these lines are long.  Can you put the longer `set_attribute` calls on multiple lines following PEP8?